### DISCUSSION TITLE
Zoom by section not drawing a section

### DISCUSSION THREAD

**Discussion by Tadza0641** [Live-Charts/LiveCharts2#1414] [Q&A]
When i do use zoom by section, it sometimes stops drawing a rectangle around the area that is supposed to be zoomed is not shown.

**Comment by guilhermeolisi** [comment]
I have the same problem in my project. Initially the rectangle is shown normally, but when I change some colors in the graph (specifically: axes, DrawMarginFrame, Legends, points and series lines) the rectangle no longer appears
